core/src/consensus/stake/mod.rs

```diff
@@ -860,7 +882,8 @@ mod tests {
         // TODO: change with stake.execute()
         self_nominate(&mut state, &address, &address_pubkey, 200, 0, 30, b"".to_vec()).unwrap();
 
-        let result = on_term_close(&mut state, 29);
+        let result = on_term_close(&mut state);
+        increase_term_id_until(&mut state, 30);
```

Comment:
If `on_term_close` handles increasing term id, these two lines will be `term_close_until(...)`